### ISSUE TITLE
Filter on 'Only translated strings' works only if the 'String contains field is empty

### ISSUE THREAD

**Issue by MagdiRizkallah** [PeaceGeeks/amani#734]
http://dev.demo1.peacegeeks.org/admin/config/regional/translate/translate
![image](https://cloud.githubusercontent.com/assets/5035201/6905193/2b7a77d6-d6db-11e4-8f65-230002b64a9a.png)

![image](https://cloud.githubusercontent.com/assets/5035201/6905212/4f3948f0-d6db-11e4-9108-b7d4aab73b48.png)

**Comment by bensinjin** [comment]
@MagdiRizkallah notice in your filter your filtering on only translated strings - there isn't a translation for social media feeds.

**Comment by MagdiRizkallah** [comment]
@bensinjin yes there is...  Quicktabs has a translated 'social media feeds' string as shown with the 'ar' not being struck-through in the 1st pic...

**Comment by bensinjin** [comment]
@MagdiRizkallah it looks like that string actually falls under the category of translated and untranslated, translated being arabic untranslated being = all langauges/language neutral/english(because thats how it was originally written) - I think it's just poorly worded

**Comment by bensinjin** [comment]
The entries with a strikethrough arabic are considered all languages

**Comment by bensinjin** [comment]
What I believe your interested in is 'all languages' => 'untranslated'

**Comment by MagdiRizkallah** [comment]
@bensinjin From my perspective, the string does appear under 'Only Translated' filter as in pic 1 above.  However, if I use the 'string contains' as additional filter, it disappears as in pic 2.
So it seems that the additional 'string contains' filter has some limitation when searching in 'Only translated strings'... 

It also seems incomplete even without the additional 'Contains' filter.  I have more translated strings than is shown in pic1.
